src/data/conversation.ts

```diff
@@ -7,10 +7,48 @@ interface ProcessResults {
   };
 }
 
+/* tslint:disable */
+// @ts-ignore
+export const SpeechRecognition =
+  // @ts-ignore
+  window.SpeechRecognition || window.webkitSpeechRecognition;
+// @ts-ignore
+export const SpeechGrammarList =
+  // @ts-ignore
+  window.SpeechGrammarList || window.webkitSpeechGrammarList;
+// @ts-ignore
+export const SpeechRecognitionEvent =
+  // @ts-ignore
+  window.SpeechRecognitionEvent || window.webkitSpeechRecognitionEvent;
+/* tslint:enable */
+
+export interface AgentInfo {
+  attribution?: { name: string; url: string };
+  onboarding?: { text: string; url: string };
+}
+
 export const processText = (
   hass: HomeAssistant,
   text: string,
   // tslint:disable-next-line: variable-name
   conversation_id: string
 ): Promise<ProcessResults> =>
-  hass.callApi("POST", "conversation/process", { text, conversation_id });
+  hass.callWS({
```

Comment:
Just make sure you don't remove the POST API at the backend, as it's used for other things like Rhasspy.

Reply:
No I kept it :-)